test/createTransform.spec.js

```diff
@@ -27,24 +26,26 @@ test('createTransform outbound', async function (t) {
   t.deepEqual(state, {a: 'hi', b: 'hi'})
 })
 
-// a terribly ugly test for inbound transforms @TODO rewrite
 test.cb('createTransform inbound', (t) => {
   const store = createStore(createReducer())
   const seedState = {a: 1, b: {b0: 2}}
 
   const inbound = (state, key) => key
   const outbound = (state, key) => 'hi'
-  let testTransform = createTransform(inbound, outbound)
-  let memoryStorage = createMemoryStorage(seedState)
-  persistStore(store, {storage: memoryStorage, transforms: [testTransform]})
 
-  setTimeout(() => {
+  const memoryStorage = createMemoryStorage(seedState)
+  const testTransform = createTransform(inbound, outbound)
+  const config = {storage: memoryStorage, transforms: [testTransform]}
+  persistStore(store, config, (err, restoredState) => {
+    t.ifError(err)
+    // t.deepEqual(restoredState, {a: 'hi', b: 'hi'})
```

Comment:
@rt2zz This assertion would be a way of implicitly testing the outbound transform.
Provided I can get this test to pass, do you think it makes sense to remove the other test in this file?

The flow of expected inbound / outbound state seems slightly more explicitly shown this way (imo).